pkg/rpcclient/notary/accounts.go

```diff
@@ -0,0 +1,36 @@
+package notary
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/nspcc-dev/neo-go/pkg/wallet"
+)
+
+// FakeSimpleAccount creates a fake account belonging to the given public key.
+// It uses a simple signature contract and this account has SignTx that
+// returns no error, but at the same time adds no signature (it obviously can't
+// do that, so CanSign() returns false for it). Use this account for Actor when
+// simple signatures are needed to be collected.
+func FakeSimpleAccount(k *keys.PublicKey) *wallet.Account {
```

Comment:
We need `FakeMultisigAccount` also, because it's allowed to combine simple and multisig signers per single main transaction. This account will have proper multisig verification script and will not be able to add a signature.

Reply:
OK, let's have it too.